client/src/plugins/element-templates-modal/modeler/PropertiesProvider.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import CamundaPropertiesProvider from 'bpmn-js-properties-panel/lib/provider/camunda/CamundaPropertiesProvider';
+
+import entryFactory from 'bpmn-js-properties-panel/lib/factory/EntryFactory';
+
+
+export default class PropertiesProvider extends CamundaPropertiesProvider {
+  constructor(eventBus, canvas, bpmnFactory, elementRegistry, elementTemplates, translate, config) {
+    super(eventBus, canvas, bpmnFactory, elementRegistry, elementTemplates, translate);
+
+    this._translate = translate;
+    this._config = config;
+
+    const getTabs = this.getTabs;
+
+    this.getTabs = element => {

```

Comment:
Is there any particular reason to declare it in the constructor?

Reply:
Can't be properly overridden because it's overridden like this in [CamundaPropertiesProvider](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/v0.35.0/lib/provider/camunda/CamundaPropertiesProvider.js#L456). 💩